cmd/containers-storage/container.go

```diff
@@ -179,6 +179,37 @@ func getContainerRunDir(flags *mflag.FlagSet, action string, m storage.Store, ar
 	return 0
 }
 
+func containerParentOwners(flags *mflag.FlagSet, action string, m storage.Store, args []string) int {
+	matched := []*storage.Container{}
+	for _, arg := range args {
+		if container, err := m.Container(arg); err == nil {
+			matched = append(matched, container)
+		}
+	}
+	if jsonOutput {
+		json.NewEncoder(os.Stdout).Encode(matched)
```

Comment:
The JSON output is a list of (details of?) containers, but that seems to be different from the `ContainerParentOwners` computation in the non-JSON case.